framework/interface/api_handlers.py

```diff
@@ -52,39 +52,34 @@ def get(self, plugin_group=None, plugin_type=None, plugin_code=None):
 class PluginNameOutput(custom_handlers.UIRequestHandler):
     SUPPORTED_METHODS = ['GET']
 
-    def get(self, target_id=None, plugin_group=None, plugin_type=None, plugin_code=None):
+    def get(self, target_id=None):
+        """Retrieve plugins details and the corresponsing data from a target scan.
```

Comment:
I found this example http://stolarscy.com/dryobates/2016-10/sphinx-rest-api/
I will start adding API documentation, when plugin is integrated with OWTF.